manifests/ocs/cluster.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+  name: rook-ceph
+  namespace: rook-ceph
```

Comment:
In order to see the ceph stats in the OCP4 console, it is required to be deployed in the 'openshift-storage' namespace